src/coreclr/gc/objecthandle.cpp

```diff
@@ -1598,11 +1607,13 @@ void Ref_UpdatePointers(uint32_t condemned, uint32_t maxgen, ScanContext* sc, Re
         HNDTYPE_STRONG,
 #ifdef FEATURE_REFCOUNTED_HANDLES
         HNDTYPE_REFCOUNTED,
-#endif
+#endif // FEATURE_REFCOUNTED_HANDLES
```

Comment:
Nit: I am not sure whether the comments on closing endif for one-liner ifdefs are an improvement. 

The comments on closing endif are useful for readability on larger ifdef/endif blocks. It is easy to see the ifdef/endif pair for one-liner, so the readability benefit is not there. In this specific case, I would say that it is even less readable with the comment since the different `HNDTYPE_` cases blend together.